keras/src/backend/torch/numpy.py

```diff
@@ -1539,6 +1539,13 @@ def split(x, indices_or_sections, axis=0):
     return list(out)
 
 
+def array_split(x, indices_or_sections, axis=0):
+    x = convert_to_tensor(x)
+    axis_int = int(axis)
```

Comment:
You already assert that axis is an `int` in the cross-backend `array_split`, so this is not needed.